<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>msvQTimePlayerWidget</class>
 <widget class="QWidget" name="msvQTimePlayerWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>513</width>
    <height>69</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Time player</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="margin">
    <number>0</number>
   </property>
   <item>
    <widget class="QToolButton" name="firstFrameButton">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="toolTip">
      <string>Go to start</string>
     </property>
     <property name="text">
      <string>&lt;&lt;</string>
     </property>
     <property name="icon">
      <iconset resource="../msvQWidgets.qrc">
       <normaloff>:/Icons/timeToFirst.png</normaloff>:/Icons/timeToFirst.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>24</width>
       <height>24</height>
      </size>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QToolButton" name="previousFrameButton">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="toolTip">
      <string>Step backward</string>
     </property>
     <property name="text">
      <string>&lt;</string>
     </property>
     <property name="icon">
      <iconset resource="../msvQWidgets.qrc">
       <normaloff>:/Icons/frameBack.png</normaloff>:/Icons/frameBack.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>24</width>
       <height>24</height>
      </size>
     </property>
     <property name="autoRepeat">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QToolButton" name="playReverseButton">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="toolTip">
      <string>Play backward / Pause</string>
     </property>
     <property name="text">
      <string>PlayBack</string>
     </property>
     <property name="icon">
      <iconset resource="../msvQWidgets.qrc">
       <normaloff>:/Icons/timePlayReverse.png</normaloff>
       <normalon>:/Icons/pause.png</normalon>:/Icons/timePlayReverse.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>24</width>
       <height>24</height>
      </size>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QToolButton" name="playButton">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="toolTip">
      <string>Play forward / Pause</string>
     </property>
     <property name="text">
      <string>Play</string>
     </property>
     <property name="icon">
      <iconset resource="../msvQWidgets.qrc">
       <normaloff>:/Icons/timePlay.png</normaloff>
       <normalon>:/Icons/pause.png</normalon>:/Icons/timePlay.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>24</width>
       <height>24</height>
      </size>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QToolButton" name="nextFrameButton">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="toolTip">
      <string>Step forward</string>
     </property>
     <property name="text">
      <string>&gt;</string>
     </property>
     <property name="icon">
      <iconset resource="../msvQWidgets.qrc">
       <normaloff>:/Icons/frameForward.png</normaloff>:/Icons/frameForward.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>24</width>
       <height>24</height>
      </size>
     </property>
     <property name="autoRepeat">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QToolButton" name="lastFrameButton">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="toolTip">
      <string>Go to end</string>
     </property>
     <property name="text">
      <string>&gt;&gt;</string>
     </property>
     <property name="icon">
      <iconset resource="../msvQWidgets.qrc">
       <normaloff>:/Icons/timeToLast.png</normaloff>:/Icons/timeToLast.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>24</width>
       <height>24</height>
      </size>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QToolButton" name="repeatButton">
     <property name="toolTip">
      <string>Repeat</string>
     </property>
     <property name="text">
      <string>Repeat</string>
     </property>
     <property name="icon">
      <iconset resource="../msvQWidgets.qrc">
       <normaloff>:/Icons/repeat.png</normaloff>:/Icons/repeat.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>24</width>
       <height>24</height>
      </size>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QDoubleSpinBox" name="speedFactorSpinBox">
     <property name="toolTip">
      <string>Time speed ratio</string>
     </property>
     <property name="prefix">
      <string>x </string>
     </property>
     <property name="decimals">
      <number>1</number>
     </property>
     <property name="minimum">
      <double>0.100000000000000</double>
     </property>
     <property name="maximum">
      <double>64.000000000000000</double>
     </property>
     <property name="singleStep">
      <double>0.200000000000000</double>
     </property>
     <property name="value">
      <double>1.000000000000000</double>
     </property>
    </widget>
   </item>
   <item>
    <widget class="ctkSliderWidget" name="timeSlider" native="true">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="decimals" stdset="0">
      <number>2</number>
     </property>
     <property name="minimum" stdset="0">
      <double>-1.000000000000000</double>
     </property>
     <property name="maximum" stdset="0">
      <double>-1.000000000000000</double>
     </property>
     <property name="value" stdset="0">
      <double>-1.000000000000000</double>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ctkSliderWidget</class>
   <extends>QWidget</extends>
   <header>ctkSliderWidget.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../msvQWidgets.qrc"/>
 </resources>
 <connections/>
</ui>
